.github/workflows/release-version.yml

```diff
@@ -0,0 +1,94 @@
+# This workflow automatically creates a release PR for the CI integration.
+
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: "Semver argument for the version bump to do on the CI integration"
+        required: true
+        default: "minor"
+        type: choice
+        options:
+          # - "patch"
+          - "minor"
+          # - "major"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      # Do the changes
+      - uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
```

Comment:
It's required to use Node.js 16 because of `ncc` (that we use in the `yarn package` script).

On newer versions of Node.js, the script fails with `ERR_OSSL_EVP_UNSUPPORTED` (cf. [related SO question](https://stackoverflow.com/questions/69719601/getting-error-digital-envelope-routines-reason-unsupported-code-err-oss)).